packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -89,6 +87,15 @@ export class KeybindingWidget extends ReactWidget {
         }
     }
 
+    protected onAfterShow(msg: Message) {
+        this.focusInputField();
+    }
+
+    protected onActivateRequest(msg: Message) {
+        super.onActivateRequest(msg);
+        this.focusInputField();
```

Comment:
Do we need to trigger `this.focusInputField();` both on `onActivateRequest` and on `onAfterShow` ?

I see the code triggered twice, in this case it is not heavy but I wonder what event we should actually use...

Reply:
thanks @marechal-p, I initially had with `onAfterShow` by following an example from the `search` widget. I noticed other widgets did not all use it so I refactored to remove it. I've re-tested and it works as intended.